config/element-api.php

```diff
@@ -841,16 +841,27 @@ function getDataPage($locale)
 {
     normaliseCacheHeaders();
 
+    $statuses = ['live', 'expired'];
+
+    /**
+     * Allow disabled versions when requesting drafts
+     * to support previews of brand new or disabled pages.
+     */
+    if (EntryHelpers::isDraftOrVersion()) {
```

Comment:
You can call this as a one-liner with `EntryHelpers::getVersionStatuses()` if you want